data/_types/index.json

```diff
@@ -3,10 +3,10 @@
     "template": "layout"
   },
   "meta": {
-    "title": "Types - UK Parliament",
+    "title": "_types.title",
```

Comment:
You will need to wrap this in a {@t key="types.title" /} and you don't need the _ before the translation key.